query-engine/js-connectors/src/conversion.rs

```diff
@@ -0,0 +1,59 @@
+use napi::bindgen_prelude::{FromNapiValue, ToNapiValue};
+use quaint::ast::Value as QuaintValue;
+use serde::Serialize;
+use serde_json::value::Value as JsonValue;
+
+#[derive(Debug, Serialize)]
+#[serde(untagged)]
+pub enum JSArg {
```

Comment:
We should agree on `JS` vs `Js` at some point (not necessary in this PR though)

Reply:
Rationale for `JS`: it's a diminutive of `JavaScript`